python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java

```diff
@@ -46,6 +46,7 @@
 import org.sonar.python.types.protobuf.SymbolsProtos;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
```

Comment:
It seems this import (as well as `org.junit.Ignore`on line 31) is unused.

Reply:
thanks, I'll remove it!